string.rb

```diff
@@ -0,0 +1,21 @@
+class String
```

Comment:
String Class is already defined in ruby so changing/adding functions to something inbuilt is not a good idea.

A much simpler thing could have been done to check inputs. Using regex. Try to write simpler code as much as possible.
```ruby
type_of_ds = gets
if type_of_ds =~ /^\d+$/
    # perform task
else
    puts "Given input is not an number"
end

#OR
def check_input_array(arr)
    arr.each { |e|
        return false if e.to_s !~ /^\d+$/
    }
    true
end
```